changelog/unreleased/bugfix-switch-columns-shared-with-me-view

```diff
@@ -1,4 +1,4 @@
-Bugfix: switch columns displayed on small screens in "Shared with me" view
+Bugfix: switch columns displayed on small screens in "Shared with me" view 
```

Comment:
Can you remove the trailing whitespace again? ;-)

Reply:
If you just want to re-trigger ci, you can simply run `git commit --amend --no-edit` to change the last commit hash and force push that.

Reply:
thanks for the advice, I indeed did it as a work around because I wasn't aware of the --no-edit command